content/committees/specification/TCKProcess.md

```diff
@@ -0,0 +1,412 @@
+---
+title: "akarta EE TCK Process 1.0
```

Comment:
There is a typo in the title and we can include a date value?

For example: 
date: '2017-09-26T14:02:37+02:00'

Reply:
I fixed the typo and added today's date.